### PR TITLE
Changing readme voice to align with comm direction

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
README

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
I had received some excellent anonymous feedback that had much to say about some pieces of our readme in the contributing section. I think we need "approachability to start from zero". Some of these simple wording changes I believe will make a huge difference.

**Does this PR introduce a breaking change?**
NO

**Other information**
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
 
 <h2 align="center">Contributing</h2>
 
-Most of the time, if webpack is not working correctly for you it is a simple configuration issue.
+**We want contributing to webpack to be fun, enjoyable, and educational for anyone, and everyone.** We have a [vibrant ecosystem](https://medium.com/webpack/contributors-guide/home) that spans beyond this single repo. We welcome you to check out any of the repositories in [this](http://github.com/webpack) organization or [webpack-contrib](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
 
-If you are still having difficulty after looking over your configuration carefully, please post
-a question to [StackOverflow with the webpack tag](https://stackoverflow.com/tags/webpack). Questions
-that include your webpack.config.js and relevant files are more likely to receive responses.
+Contributions go far beyond pull requests and commits. Although we love giving you the opportunity to put your stamp on webpack, we also are thrilled to recieve a variety of other contributions including:
```
Comment (skipjack) [inline README.md]: s/recieve/receive

File: README.md [hunk 2]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
...
+_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!! 
+
+<h3 align="center">Creating your own plugins and loaders</h3>
+If you create a loader or plugin, we would <3 for you to open source it, and put it on npm. We follow the `x-loader`, `x-webpack-plugin` naming convention.
+
+<h2 align="center">Support</h2>
+
+We consider webpack to be a low-level tool that is used by not only individuals but also layered beneath lots of other awesome tools. Because of this, we know that sometimes webpack is not the _easiest_ to configure, and we constantly are working on finding ways to improve this!!!
+
+If your just getting started, take a look at [our new docs and concepts page](webpack.js.org/concepts). This has a high level overview that is great for beginners!!
```
Comment (skipjack) [inline README.md]: s/your/you're

**Comment by TheLarkInn** [comment]
@skipjack One more review? I changed some tiny things.

File: README.md [hunk 3]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
 
 <h2 align="center">Contributing</h2>
 
-Most of the time, if webpack is not working correctly for you it is a simple configuration issue.
+**We want contributing to webpack to be fun, enjoyable, and educational for anyone, and everyone.** We have a [vibrant ecosystem](https://medium.com/webpack/contributors-guide/home) that spans beyond this single repo. We welcome you to check out any of the repositories in [our](http://github.com/webpack) organization or [webpack-contrib](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
```
Comment (skipjack) [inline README.md]: I think "organization" should go within the `[our](...)` link e.g. `[our organization](...)`. Also maybe add "organization" after `[webpack-contrib]` as well. So with both changes:

``` md
... We welcome you to check out any of the repositories in [our organization](http://github.com/webpack) or the [webpack-contrib organization](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
```

File: README.md [hunk 4]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
...
+* Adding unit, or functional tests
+
+* Triaging GitHub issues (especially helping find out if an issue still persists or is reproducible)
+
+* [Searching #webpack on twitter](https://twitter.com/search?q=webpack) and helping someone else who needs help
+
+* Teaching others how to contribute to one of the many webpack repo's!
+
+* [Blogging, speaking about, or creating tutorials](https://github.com/webpack-contrib/awesome-webpack) about one of webpack's many features. 
+
```
Comment (skipjack) [inline README.md]: This is somewhat nitpicky but you may want to remove the line breaks between list items to stay consistent with the rest of the doc (also I'm not sure all md processors will handle this as one list).

File: README.md [hunk 5]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
...
-on npm and following the `x-loader`, `x-webpack-plugin` convention.
+* Spelling or grammar fixes
 
-You are also welcome to correct any spelling mistakes or any language issues.
+* README.md corrections or redesigns
 
-If you want to discuss something or just need help, [here is our Gitter room](https://gitter.im/webpack/webpack).
+* Adding unit, or functional tests
+
+* Triaging GitHub issues (especially helping find out if an issue still persists or is reproducible)
```
Comment (skipjack) [inline README.md]: Maybe better as:

``` md
Triaging GitHub issues -- especially determining whether an issue still persists or is reproducible.
```

File: README.md [hunk 6]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
...
+
+* Teaching others how to contribute to one of the many webpack repo's!
+
+* [Blogging, speaking about, or creating tutorials](https://github.com/webpack-contrib/awesome-webpack) about one of webpack's many features. 
+
+* Helping others in our webpack [gitter channel](https://gitter.im/webpack/webpack).
+
+If you are worried or don't know where to start, you can **always** reach out to [Sean Larkin (@TheLarkInn) on Twitter](https://twitter.com/thelarkinn) or simply submit an issue and a maintainer can help give you guidance! 
+
+We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributors Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help.
```
Comment (skipjack) [inline README.md]: s/Contributors/Contributor's

File: README.md [hunk 7]
```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
...
+We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributors Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help. 
+
+_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!! 
+
+<h3 align="center">Creating your own plugins and loaders</h3>
+If you create a loader or plugin, we would <3 for you to open source it, and put it on npm. We follow the `x-loader`, `x-webpack-plugin` naming convention.
+
+<h2 align="center">Support</h2>
+
+We consider webpack to be a low-level tool that is used by not only individuals but also layered beneath lots of other awesome tools. Because of this, we know that sometimes webpack is not the _easiest_ to configure, and we constantly are working on finding ways to improve this!!!
```
Comment (skipjack) [inline README.md]: I would rephrase slightly:

``` md
We consider webpack to be a low-level tool used not only individuals but also layered beneath other awesome tools. Because of it's flexibility, webpack isn't always the _easiest_ entry-level solution, however we do believe it is the most powerful. That said, we're always looking for ways improve and simplify the tool without compromising functionality. If you have any ideas on ways to accomplish this, we're all ears!
```

**Comment by skipjack** [comment]
> One more review? I changed some tiny things.

Yep I had a bit more time on this one and added some comments for things I missed last time.

**Comment by sokra** [comment]
@skipjack Can this be merged?

**Comment by skipjack** [comment]
@sokra idk if @TheLarkInn had a chance to see the second review. You could always merge and then make these changes in a separate PR or ignore them if you don't think they're important. If I had access to this repo I'd be happy to commit the changes directly on this PR.

**Comment by TheLarkInn** [comment]
@skipjack Alright made all suggested changes.
